ui/Screen/Onboarding/Success.svelte

```diff
@@ -45,8 +45,9 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your handle — it's unique to you!<br /> Click to copy it and share
-      with others so that they can find you.
+      This is a <span class="typo-text-bold">peer address</span>! It's unique to
+      you and this computer. You can find it at any time in your Settings page.
+      You'll need to share it with others to collaborate.
     </p>
```

Comment:
We should also change the code above to actually show a `PeerId` instead of our own `URN`.